flask_debugtoolbar/__init__.py

```diff
@@ -50,6 +50,7 @@ class DebugToolbarExtension(object):
         os.path.join(os.path.dirname(__file__), 'static'))
 
     _redirect_codes = [301, 302, 303, 304]
+    _success_codes = [200, 201]
```

Comment:
Can you provide a little more background on this use case? Why do you want the toolbar visible on a `201`? Is it for the sql pane?

Reply:
This was mainly for convenient access to log entries generated while debugging, though the SQL and profiler panes may come in handy at some point in the future.